packages/devextreme/js/__internal/ui/map/m_provider.dynamic.azure.ts

```diff
@@ -48,7 +48,12 @@ class AzureProvider extends DynamicProvider {
       driving: 'car',
       walking: 'pedestrian',
     };
-    return movementTypes[type] || movementTypes.driving;
+
+    if (!isDefined(type)) {
+      return movementTypes.driving;
+    }
+
+    return movementTypes[type] || type;
```

Comment:
Minor suggestion, this will be semantically more correct
```suggestion
    return movementTypes[type] ?? type;
```

Reply:
updated